src/database/models/user.js

```diff
@@ -65,5 +65,11 @@ module.exports = (sequelize, DataTypes) => {
       as: 'articleId'
     });
   };
+  User.associate = models => {
+    User.hasMany(models.Article, {
+      foreignKey: 'authorId',
+      as: 'userArticles'
```

Comment:
Please check use `Articles` here as some eagerload methods already depend on it

Reply:
ok